blazor/pager/getting-started.md

```diff
@@ -248,9 +248,67 @@ For Blazor WebAssembly App, refer script in the `<head>` of the **~/index.html**
 {% endhighlight %}
 {% endtabs %}
 
-* Now, add the Syncfusion Blazor Pager component in razor file. Here, the Pager component is added in the **~/Pages/Index.razor** file under the **~/Pages** folder.
+Here, the Pager component is integrated with the ListView component. So, first ListView component is added in the **~/Pages/Index.razor** file under the **~/Pages** folder. Refer [here](https://blazor.syncfusion.com/documentation/listview/getting-started) for create the Syncfusion ListView component.
```

Comment:
Here, the Pager component is integrated with the ListView component. So, the first ListView component is added in the **~/Pages/Index.razor** file under the **~/Pages** folder. Refer [here](https://blazor.syncfusion.com/documentation/listview/getting-started) to create the Syncfusion ListView component.